### PR TITLE
fix various table issues and dependent text

### LINKED ISSUE #1595: Requirements table for canvas.label is wrong
As found by @tomcrane, the requirements table in Prezi3 says that `label` is required for Canvas, which is no longer correct.

### LINKED ISSUE #1608: Appendix A tables have 99 problems, but being useful ain't one
- [x] The new images from prezi3 are gone. Probably they needed to go to IIIF/website?
- [x] Canvas SHOULD have `label`
- [x] Range SHOULD have `label`
- [x] AnnotationCollection SHOULD have `label`
- [x] Merge Image and Other Content (identical requirements)
- [x] Canvas MAY have `thumbnail` is the primary requirement
- [x] `navDate` is under descriptive now
- [x] Range MAY have `navDate`
- [x] Canvas MAY have `navDate`
- [x] Annotation MUST have `id`
- [x] AnnotationPage SHOULD have `id` but see #1607 
- [x] `language` is missing from the tables completely
- [x] As is `profile`
- [x] As is `duration`
- [ ] Canvas `height`, `width` and `duration` are ... complicated
- [x] Collection MAY have `viewingDirection`
- [x] AnnotationCollection MUST NOT have `viewingDirection`
- [x] `timeMode` is missing from the tables
- [x] `logo` is now a linking property
- [x] Manifest MAY have `start`
- [x] `supplementary` is missing from the tables
- [x] Exceptions should be noted with a backlink to the right section 
- [x] Deref table is probably worthless. If it's entirely embedded, it MAY be also dereffed. If it's not, then it MUST be dereffed. Should fix in the prose.

Thanks to @jronallo for bringing up the first few issues that led to the extensive list above.

### LINKED ISSUE #1618: Remove Protocol Behavior table
... and explain in the prose.
(from #1608): 

> Deref table is probably worthless. If it's entirely embedded, it MAY be also dereffed. If it's not, then it MUST be dereffed. Should fix in the prose.

### LINKED ISSUE #1620: Metadata requirements tables are now very wide
Which generates horizontal scroll bars at a fairly wide viewport. Split into more tables?

e.g., "Technical properties, continued"

That's a bit messy.

### PR DESCRIPTION
**Pull request by tomcrane**
Closes #1620 ; Closes #1618 ; Closes #1608 ; Closes #1595

### CODE REVIEW

**Comment by tomcrane** [comment]
Not sure about the footnotes to the two tables. There's a spacing issue with them too.
Also, these tables have placeholderCanvas and accompanyingCanvas in the tables, but the defns of those not merged yet.

File: source/api/presentation/3.0/index.md [hunk 1]
```diff
@@ -342,7 +342,7 @@ The URI that identifies the resource. If the resource is only available embedded
 
 The value _MUST_ be a string, and the value _MUST_ be an HTTP(S) URI for resources defined in this specification. If the resource is retrievable via HTTP(S), then the URI _MUST_ be the URI at which it is published. External resources, such as profiles, _MAY_ have non-HTTP(S) URIs defined by other communities.
 
-The existence of an HTTP(S) URI in the `id` property does not mean that the URI will always be dereferencable. The [definitions of the Resources][prezi30-resource-structure] give guidance as to this, as does the [table in Appendix A][prezi30-appendixa]. 
+The existence of an HTTP(S) URI in the `id` property does not mean that the URI will always be dereferencable.  If the resource with the `id` property is __embedded__, it _MAY_ also be dereferenceable. If the resource is __referenced__, it _MUST_ be dereferenceable. The [definitions of the Resources][prezi30-resource-structure] give further guidance.
```
Comment (azaroth42) [inline source/api/presentation/3.0/index.md]: :+1:

File: source/api/presentation/3.0/index.md [hunk 2]
```diff
@@ -1515,38 +1515,53 @@ It is possible to include Image API service descriptions within the Manifest, an
...
+| AnnotationCollection | ![recommended][icon3-rec] | ![optional][icon3-opt]    | ![optional][icon3-opt]    | ![optional][icon3-opt] | ![optional][icon3-opt] | ![not allowed][icon3-na]  | ![not allowed][icon3-na]  |
+| Content Resources    | ![optional][icon3-opt]    | ![optional][icon3-opt]    | ![optional][icon3-opt]    | ![optional][icon3-opt] | ![optional][icon3-opt] | ![not allowed][icon3-na]  | ![recommended][icon3-rec] |
 {: .api-table #table-reqs-1}
 
+|                      | thumbnail                 | placeholderCanvas        | accompanyingCanvas       |
+| -------------------- | ------------------------- | ------------------------ | ------------------------ | 
+| Collection           | ![recommended][icon3-rec] | ![optional][icon3-opt]   | ![optional][icon3-opt]   |
+| Manifest             | ![recommended][icon3-rec] | ![optional][icon3-opt]   | ![optional][icon3-opt]   |
+| Canvas*               | ![optional][icon3-opt]    | ![optional][icon3-opt]   | ![optional][icon3-opt]   |
```
Comment (azaroth42) [inline source/api/presentation/3.0/index.md]: Maybe put the * in the cell, rather then the row header?

File: source/api/presentation/3.0/index.md [hunk 3]
```diff
@@ -1515,38 +1515,53 @@ It is possible to include Image API service descriptions within the Manifest, an
...
+| Range                 | ![required][icon3-req]  | ![required][icon3-req] | ![not allowed][icon3-na] | ![not allowed][icon3-na] | ![not allowed][icon3-na] | ![not allowed][icon3-na] | ![not allowed][icon3-na]  | ![optional][icon3-opt]   | ![optional][icon3-opt] | ![not allowed][icon3-na] |
+| Annotation Collection | ![required][icon3-req]  | ![required][icon3-req] | ![not allowed][icon3-na] | ![not allowed][icon3-na] | ![not allowed][icon3-na] | ![not allowed][icon3-na] | ![not allowed][icon3-na]  | ![not allowed][icon3-na] | ![optional][icon3-opt] | ![not allowed][icon3-na] |
+| Content Resources     | ![required][icon3-req]  | ![required][icon3-req] | ![optional][icon3-opt]   | ![optional][icon3-opt]   | ![optional][icon3-opt]   | ![optional][icon3-opt]   | ![optional][icon3-opt]    | ![not allowed][icon3-na] | ![optional][icon3-opt] | ![not allowed][icon3-na] |
 {: .api-table #table-reqs-2}
 
+*If a Canvas has either of `height` and `width`, it _MUST_ have the other, as described in the [definitions][prezi30-height] of those properties.<br/>
```
Comment (azaroth42) [inline source/api/presentation/3.0/index.md]: c.f. previous * comment

Comment (tomcrane) [reply source/api/presentation/3.0/index.md]: pushed... looks a little odd like that too.

**Comment by tomcrane** [comment]
Also, I've split the first table.

**Comment by azaroth42** [comment]
Oh, no preview link because travis fails due to unrelated stuff. :(

**Pull request merged**